clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -57,7 +57,7 @@ private URI translateUri(URI uri) throws java.net.URISyntaxException {
                 return new URI("s3a", uri.getUserInfo(), uri.getHost(), uri.getPort(), uri.getPath(), uri.getQuery(),
                         uri.getFragment());
             default:
-                throw new RuntimeException(String.format("unsupported URI scheme %s", uri.getScheme()));
+                throw new RuntimeException(String.format("unsupported URI scheme %s, lakeFS file-system currently supports s3a only", uri.getScheme()));
```

Comment:
This message may be a bit confusing, as the URI scheme returned from lakeFS will be `s3`.  (E.g. consider if you managed to convince the lakeFS server to return URI scheme `s3a`.  This code would throw a message "unsupported URI scheme s3a, lakeFS file-system currently supports s3a only"...).

Maybe something like this?
```suggestion
                throw new RuntimeException(String.format("cannot translate URI scheme %s from lakeFS server, lakeFS FileSystem currently supports translating s3 => s3a only", uri.getScheme()));
```

Reply:
Translation part might confuse the users, adding the second part and keeping the `Unsupported` part explicit.